### PR TITLE
fix(Firma con IO): [SFEQS-1342] Update abort signature flow bottom sheet copy

### PR DESCRIPTION
**Pull request by LazyAfternoons**
## Short description
This PR updates the FCI copy of the abort signature flow bottom sheet.

## List of changes proposed in this pull request
- Updates the locale for both IT and EN translation;
- Adds a selector for the signature `dossier_title` parameter; 
- Edits the bottom sheet string parsed by the `Markdown` component.

## How to test
- Start the dev server with the following configuration: 
```
...
fci: {
      waitForSignatureCount: 1,
      rejectedCount: 0,
      expiredCount: 0,
      expired90Count: 0,
      waitForQtspCount: 0,
      signedCount: 0,
      noSignatureFieldsCount: 0
    },
...
```
- Run `yarn generate`;
- Start the app by pointing to the dev server and start a signature flow from the message; 
- Press `Go to the documents` and then press `Cancel`;
- The bottom sheet copy should be updated accordingly.

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -3215,7 +3215,7 @@ features:
           subTitle: The signature request is not associated with your Tax Code.
     abort:
       title: "Quit signing"
-      content: "Do you want to quit signing the documents?"
```
Comment (LazyAfternoons) [inline locales/en/index.yml:3218]: cc @AliceAzzolini

File: locales/en/index.yml [hunk 2]
```diff
@@ -3215,7 +3215,7 @@ features:
...
+      content: "Do you want to quit signing **{{dossierTitle}}** documents?"
```
Comment (LazyAfternoons) [inline locales/en/index.yml:3218]: cc @AliceAzzolini

File: ts/features/fci/store/reducers/fciSignatureRequest.ts [hunk 3]
```diff
@@ -94,4 +94,15 @@ export const fciSignatureRequestIdSelector = createSelector(
     pot.isSome(signatureDetailView) ? signatureDetailView.value.id : undefined
 );
 
+/**
+ * Selects the dossier title if present, undefined otherwise.
+ */
+export const fciSignatureRequestDossierTitleSelector = createSelector(
```
Comment (Vangaorth) [inline ts/features/fci/store/reducers/fciSignatureRequest.ts:100]: Do we need to cache this selector? It seems to be a direct read that access an already in-memory data

Comment (LazyAfternoons) [reply ts/features/fci/store/reducers/fciSignatureRequest.ts]: Thanks for the suggestion, we indeed don't need memoization there.

**Review by AliceAzzolini — APPROVED** [review]
Updated the English version of the bottomsheet, the Italian one looks good as it is!

**Review by Vangaorth — APPROVED** [review]
LGTM!

**Review by hevelius — APPROVED** [review]
LGTM!

**Pull request merged**